rules/integrations/okta/credential_access_attempted_bypass_of_okta_mfa.toml

```diff
@@ -58,6 +58,8 @@ references = [
     "https://developer.okta.com/docs/reference/api/system-log/",
     "https://developer.okta.com/docs/reference/api/event-types/",
     "https://www.elastic.co/security-labs/testing-okta-visibility-and-detection-dorothy",
+    "https://www.elastic.co/security-labs/monitoring-okta-threats-with-elastic-security",
```

Comment:
I'd add these to all Okta, along with https://www.elastic.co/security-labs/starter-guide-to-understanding-okta